dev/gen_certs.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+set -e
+
+devdir="$(
+  cd "${0%/*}" 2>/dev/null
+  echo "$PWD"
+)"
+mkdir -p ${devdir}/certs
+
+if [ ! -z "$1" ]; then
+  HOST=$1
+else
+  HOST=$(hostname -f)
+fi
+
+if [ ! -e "${devdir}/certs/ca-cert.pem" ]; then
+  cd ${devdir}/certs
+  echo "Make sure to use different values for the Common Name (FQDN) in CA certificate and certificate."
+
+  echo "Generate CA Certificate:"
+  openssl genrsa 2048 >ca-key.pem
+  openssl req -new -x509 -nodes -days 3650 -key ca-key.pem -out ca-cert.pem -subj "/CN=couch"
+
+  echo "Generate Server Certificate:"
+  openssl req -newkey rsa:2048 -nodes -days 3650 -keyout s-key.pem -out s-req.pem -subj "/CN=$HOST"
+  openssl x509 -req -days 3650 -set_serial 01 -in s-req.pem -out s-cert.pem -CA ca-cert.pem -CAkey ca-key.pem
```

Comment:
If these are the per-node certificates I think we only need one certificate (the node is a client and a server depending on viewpoint) with CN of erlang node name